### PR TITLE
2.11.3 staging merge

### PR DESCRIPTION
**Pull request by selinaarokiaswamy-GSLAB**

### CODE REVIEW

**Comment by selinaarokiaswamy-GSLAB** [comment]
@JosephPJordan could you please review and merge this PR to staging?

File: docs/pre-req.rst [hunk 1]
```diff
@@ -32,4 +32,4 @@ Pre-requisites for the F5 ACI ServiceCenter:
   | 2.11, 2.11.x                | 3.45.0+                | 1.17.0+                        |  1.9.1+                               |
   +-----------------------------+------------------------+--------------------------------+---------------------------------------+
 
-Note:  It is recommended to always install the latest plugin version on the BIG-IP device.
+Note:  It is **recommended** to always install the **latest** plugin version on the BIG-IP device.
```
Comment (JosephPJordan) [inline docs/pre-req.rst:35]: I suggest changing this to either:
- F5 recommends always installing the **latest** plugin version on the BIG-IP device.
or
- We recommend always installing the **latest** plugin version on the BIG-IP device.

**Review by JosephPJordan — CHANGES_REQUESTED** [review]
One suggested change.  Sorry for the delay, was off the last couple days.

**Comment by selinaarokiaswamy-GSLAB** [comment]
@JosephPJordan I have included your suggestion in the Pre-requisites file. Could you please merge the changes?

**Review by JosephPJordan — APPROVED** [review]

**Pull request merged**